opennms-config/src/main/java/org/opennms/netmgt/filter/JdbcFilterDao.java

```diff
@@ -83,6 +87,16 @@
 	private DataSource m_dataSource;
     private DatabaseSchemaConfig m_databaseSchemaConfigFactory;
 
+    private final static MetricRegistry metricRegistry = new MetricRegistry();
+    private final JmxReporter jmxReporter;
+    private final Timer getIpListTimer;
+
+    public JdbcFilterDao() {
+        getIpListTimer = metricRegistry.timer("getIPAddressList");
+        jmxReporter = JmxReporter.forRegistry(metricRegistry).inDomain("org.opennms.netmgt.config").build();
```

Comment:
Let's register the reporter when the bean is initialized (see org.springframework.beans.factory.InitializingBean) and remove it when the bean is destroyed (see org.springframework.beans.factory.DisposableBean) instead if just registering the constructor.